mandatory/4-tax.js

```diff
@@ -17,7 +21,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let productPrice = calculateSalesTax(price);
+  return "£" + productPrice.toFixed(2);
+}
```

Comment:
Consider: function addTaxAndFormatCurrency(sales) {
return "£" + calculateSalesTax(sales).toFixed(2);
}

Reply:
Thank you for your review and comment.